web3swift/src/Account/EthereumAccount.swift

```diff
@@ -38,9 +38,9 @@ public class EthereumAccount: EthereumAccountProtocol {
         return KeyUtil.generateAddress(from: self.publicKeyData)
     }()
     
-    required public init(keyStorage: EthereumKeyStorageProtocol, keystorePassword password: String) throws {
+    required public init(address: String, keyStorage: EthereumKeyStorageProtocol, keystorePassword password: String) throws {
```

Comment:
Would prefer `account` API not to change for two reasons:
- Generating a key + creating account without caring about public address is a valid usecase (in this case would need to initialize with address parameter)
- `EthereumAccount` only cares about 1 account. Storage having more than one key is a detail.

For this reason I think it's better to have 2 protocols for key storage:
`EthereumKeyStorageProtocol` (same as current)
`EthereumMultipleKeyStorageProtocol` (new)

Can add a new initializer using `EthereumMultipleKeyStorageProtocol` and `address` parameter (but using `EthereumAddress` instead of `String`), and another method for `create` too.